images/linux/scripts/installers/clang.sh

```diff
@@ -41,14 +33,16 @@ function SetDefaultClang {
 wget https://apt.llvm.org/llvm.sh
 chmod +x llvm.sh
 
+toolset="$INSTALLER_SCRIPT_FOLDER/toolset.json"
+
 if isUbuntu16 || isUbuntu18; then
-   versions=( "6.0" "8" "9" )
-   default_clang_version="9"
+    versions=$(jq -r '.Clang.versions[]' $toolset)
+    default_clang_version=$(jq -r '.Clang.default_version' $toolset)
 fi
 
 if isUbuntu20 ; then
-    versions=( "8" "9" "10" )
-    default_clang_version="10"
+    versions=$(jq -r '.Clang.versions[]' $toolset)
+    default_clang_version=$(jq -r '.Clang.default_version' $toolset)
 fi
```

Comment:
Why do we need these conditions?
Code is the same for all platforms

Reply:
Thank you, fixed now